packages/sw-runtime-caching/src/lib/network-first.js

```diff
@@ -53,6 +59,10 @@ class NetworkFirst extends Handler {
   constructor(input = {}) {
     super(input);
 
+    const cacheableResponse = new CacheableResponse({statuses: [0, 200]});
+    this.requestWrapper.cacheableResponseCheck =
+      cacheableResponse.isResponseCacheable.bind(cacheableResponse);
+
```

Comment:
It seems a little strange that you need to bind a cacheableResponse's entry to...the parent? Could you explain what's going on here?